core/src/apps/management/recovery_device/layout.py

```
@@ -321,30 +321,30 @@ def __init__(self, msg: str) -> None:
             if (blob := await ctx.read(key="mnemonic")) is None:
-                raise RetryRead(TR.n4w1__err_empty)
+                raise RetryRead(TR.n1w1__err_empty)
 
-            # TODO(N4W1): use protobuf?
+            # TODO(N1W1): use protobuf?
             blob = bytes(blob)
             try:
                 return blob.decode()
             except ValueError:
-                raise RetryRead(TR.n4w1__err_damaged)
+                raise RetryRead(TR.n1w1__err_damaged)
```

Ruff (0.16.1)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)